docs/release-notes/8.8.asciidoc

```diff
@@ -1,6 +1,41 @@
 [[release-notes-header-8.8.0]]
 == 8.8
 
+[discrete]
+[[release-notes-8.8.1]]
+=== 8.8.1
+
+[discrete]
+[[breaking-changes-8.8.1]]
+==== Breaking changes
+
+//tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+// THIS ALSO MEANS IF YOU USE LINKS HERE, THEY SHOULD BE FULL URLS WITH NO ATTRIBUTES
+
+There are no breaking changes in 8.8.1.
+
+//end::breaking-changes[]
+
+[discrete]
+[[features-8.8.1]]
+==== New features
+
+* Introduces the {kibana-ref}/gen-ai-action-type.html[Generative AI connector] and Security Assistant for {elastic-sec} ({pull}157228[#157228] {pull}156933[#156933]).
+
+[discrete]
+[[bug-fixes-8.8.1]]
+==== Bug fixes and enhancements
+* Fixes an Anomalies table bug that left {ml} jobs greyed out after they were installed ({pull}158821[#158821]).
+* Fixes an Anomalies table bug that stopped some job counts from being shown ({pull}158739[#158739]).
```

Comment:
We should prob combine these into one since it's the same feature. 
```suggestion
* Fixes bugs in the Anomalies table that left {ml} jobs greyed out after they were installed ({pull}158821[#158821]), and that stopped some job counts from being shown ({pull}158739[#158739]).
```